src/app/examples/examples/examples.component.scss

```diff
@@ -3,6 +3,7 @@
 nav {
   margin-bottom: 20px;
   position: relative;
+  overflow: scroll;
```

Comment:
This seems to be unfortunately  broken in Google Chrome...
![anms-error](https://user-images.githubusercontent.com/3764868/45807826-9b4e1e00-bcc4-11e8-8825-7ea7b39caa32.png)